group_vars/all.yml

```diff
@@ -173,3 +173,9 @@ shift_stack: false
 #      ram: 16384
 #      disk: 40
 #      vcpus: 4
+
+# set os_version that is needed depending on the RHOSP minor version
+#Also, in case of scale lab please check if the rhel version is available
+force_os_install: false
+#os_version: RHEL 7.7
+
```

Comment:
allow user to set `os_install: RHEL 8.2` here (you can change the variable name if you want)

Reply:
personally not ok to allow the user to set the OS version not sure everyone aware of the OSP release vs RHEL release and it adds one more extra variable in the group vars.

Reply:
- If the user does not set ``force_os_install: true`` then the undercloud host is reimaged based on the OSP release. (default way)
- But if the user wants to use jetpack to deploy OSP for minor release this PR adds an option to specify the os_version. (optional way)  as requested by @pradiptapks

Reply:
ack, how about implementing as suggested in my first comment?

Reply:
Done